<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RECORD MEETING</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LM Roman 12</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RECORD MEETING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_meeting_btn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE MEETING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SETTINGS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>30</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>66</y>
     <width>1371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MEETING ID:   0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DATE</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="meeting_date">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>150</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TIME STARTED</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="meeting_time_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TIME ENDED</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_ended">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FACILITATOR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="facilitator_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AGENDA</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="agenda">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>15</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>PROPOSALS AND DISCUSSIONS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_proposal_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD NEW</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>581</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>20</width>
     <height>691</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>751</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ATTENDANCE</string>
    </attribute>
    <widget class="QLabel" name="checked_count_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TOTAL ATTENDANCE :     35 OUT OF 50</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>731</width>
       <height>471</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>241</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ATTENDEND</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ON TIME</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LATE PENALTY: KSH. 100</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>383</x>
       <y>0</y>
       <width>20</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MISSED PENALTY: KSH. 300</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mark_all_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Mark All</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CONTRIBUTIONS</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>731</width>
       <height>521</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>241</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AMOUNT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PAID BY</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="total_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TOTAL CONTRIBUTION:     KSH. XXXX</string>
     </property>
    </widget>
    <widget class="QLabel" name="req_amount">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REQUIRED AMOUNT IN KSH</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>6</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="meeting_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>LABEL / NAME THE MEETING (OPTIONAL)</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
